login/views.py

```diff
@@ -0,0 +1,47 @@
+from django.shortcuts import redirect, render
+from django.http import HttpResponse
+
+from .models import User
+from .forms import LoginForm
+
+#variables
+user_index = 0
+error_message = ''
+
+def change_user_index(number):
+    user_index = number
+
+# Create your views here
+def account_view(request):
+    if user_index != 0:
+        user = User.objects.get(id=user_index)
+        context = {
+            'user_index': user_index,
+            'current_user': user
+        }
+    else:
+        context = {
+            'user_index': user_index
+        }
+    return render(request, "account/account.html", context)
+
+def login_view(request):
```

Comment:
I kinda don't like the use of global variables in this case. I think the best alternative is to use something called django sessions. 

https://docs.djangoproject.com/en/4.0/topics/http/sessions/

Reply:
Reason for this is that other modules can access values from the session too. So if we are to have a personalized instance for a person with their own inventory of ingredients, then we would need to pass it to the session